app/templates/index.hbs

```diff
@@ -95,7 +95,7 @@
                             {{t "index.services.bottom.div.line3"}}
                         </div>
                     </p>
-                    <a href="mailto:contact@osf.io" class="btn btn-info btn-lg" onclick={{action "click" "link" "Preprints - Index - Contact"}}>{{t "index.services.bottom.contact"}}</a>
+                    <a href="mailto:contact@osf.io" class="btn btn-info btn-lg" onbeforeclick={{action "click" "link" "Preprints - Index - Contact"}}>{{t "index.services.bottom.contact"}}</a>
```

Comment:
This was a temporary fix. It is addressed now in pull request for ticket EOSF-178.